torch/testing/_internal/common_utils.py

```diff
@@ -829,8 +829,13 @@ def assertTensorsEqual(a, b):
                                 self.assertTrue(torch.equal(inf_sign, torch.isinf(b).sign()), message)
                                 diff[inf_mask] = 0
                         # TODO: implement abs on CharTensor (int8)
-                        if diff.is_signed() and diff.dtype != torch.int8:
+                        # TODO: modify abs to return float/double for ComplexFloat/ComplexDouble
+                        if diff.dtype.is_complex or (diff.is_signed() and diff.dtype != torch.int8):
```

Comment:
Shouldn't complex numbers report that they are signed? Or am I missing something? :)

Reply:
std::numeric_limits, which is what we use in is_signed function, doesn't support non-standard arithmetic types such as complex<T>. we can however decide to have is_signed() for complex numbers which will always return true. thoughts?